cves/kernel/CVE-2020-15393.yml

```diff
@@ -151,10 +151,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: This bug was found using FuzzUSB, a fuzzer for USB gadget stacks within OS kernels. Info on that tool is at https://ieeexplore.ieee.org/document/9833593
```

Comment:
similarly, consider splitting into multiple lines for readability, good context

Reply:
As above, I figured out a way to do it in YAML so I'll add newlines.